validator/libs/checks.js

```diff
@@ -175,71 +175,69 @@ module.exports = {
   //check if an example is supported by contextBroker
   exampleSupported: function(fullPath) {
     var check = true;
-    var apiInstance = getApiEntityClient();
+    var apiInstance=getApiEntityClient();
```

Comment:
any reason to remove the spaces? if linter asked for that it looks strange.

Reply:
don't know the reason why it happened